src/Compiler/Compiler.h

```diff
@@ -141,7 +140,8 @@ class Compiler {
   virtual void Help(std::ostream* out);
   virtual void Version(std::ostream* out);
   virtual void Message(const std::string& message,
-                       const std::string& messagePrefix = "") const;
+                       const std::string& messagePrefix = "",
+                       const bool raw = false) const;
```

Comment:
Please update `ErrorMessage` as well and replace `const bool raw = false` with `bool raw = false`. const doesn't make sense in this case.

Reply:
Ok.I will remove it.

Reply:
Please update `ErrorMessage` as well for consistency

Reply:
No problem.